common/persistence/serializer_test.go

```diff
@@ -359,6 +374,18 @@ func (s *cadenceSerializerSuite) TestSerializer() {
 			s.Nil(err)
 			s.NotNil(resetPointsEmpty)
 
+			isolationGroupConfiguration0Thrift, err := serializer.SerializeIsolationGroups(isolationGroupCfg0, common.EncodingTypeThriftRW)
```

Comment:
the order of the tests is incorrect, and you only tested thrift encoding and decoding